drivers/docker/fingerprint.go

```diff
@@ -167,18 +166,7 @@ func (d *Driver) buildFingerprint() *drivers.Fingerprint {
 		fp.Attributes["driver.docker.runtimes"] = pstructs.NewStringAttribute(
 			strings.Join(runtimeNames, ","))
 		fp.Attributes["driver.docker.os_type"] = pstructs.NewStringAttribute(dockerInfo.OSType)
-
-		if runtime.GOOS == "windows" && dockerInfo.OSType == "linux" {
-			if d.fingerprintSuccessful() {
-				d.logger.Warn("detected Linux docker containers on Windows; only Windows containers are supported")
-			}
-
-			d.setFingerprintFailure()
-			return &drivers.Fingerprint{
-				Health:            drivers.HealthStateUnhealthy,
-				HealthDescription: "Docker is configured with Linux containers; only Windows containers are supported",
-			}
-		}
+		fp.Attributes["driver.docker.kernel_version"] = pstructs.NewStringAttribute(dockerInfo.KernelVersion)
```

Comment:
This addition looks fine, but it's not clear to me why we've included it in this PR? Is there something we need from this attribute?

Reply:
You are right, it was a leftover from the development, removing it right now.